storage/BUILD

```diff
@@ -24,7 +23,10 @@ cc_library(
     copts = ["-DLOGGER_ENABLE"],
     linkopts = ["-lcassandra"],
     deps = [
-        "//accelerator:common_core",
         "//accelerator:ta_errors",
+        "//utils:ta_logger",
+        "@com_github_uthash//:uthash",
+        "@entangled//common/model:bundle",
+        "@entangled//utils/containers/hash:hash243_queue",
```

Comment:
May I ask why you include `"@entangled//common/model:bundle",`, `"@entangled//utils/containers/hash:hash243_queue",` here?

Reply:
The previous dependency leads to an error : cycle in dependency graph
"@entangled//common/model:bundle" if for `NUM_FLEX_TRITS_HASH` and other definitions.
"@entangled//utils/containers/hash:hash243_queue" is for hash_queue in `scylla_api.h`, I will move it to bazel rule `storage`